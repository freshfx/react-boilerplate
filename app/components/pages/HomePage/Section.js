import styled from '@emotion/styled'

const Section = styled.section`
  margin: 3em auto;

  &:first-of-type {
    margin-top: 0;
  }
`

export default Section
