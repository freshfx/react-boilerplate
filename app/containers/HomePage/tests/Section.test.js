import 'jest-styled-components'
import React from 'react'
import {mount} from 'enzyme'

import Section from '../Section'

const renderComponent = (props = {}) => mount(<Section {...props} />)

describe('<Section />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderComponent()
    expect(renderedComponent).toMatchSnapshot()
  })
})

