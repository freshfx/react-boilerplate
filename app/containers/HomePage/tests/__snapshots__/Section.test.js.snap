// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> should match snapshot 1`] = `
.c0 {
  margin: 3em auto;
}

.c0:first-child {
  margin-top: 0;
}

<Section>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "Section-sc-1eulrsm-0",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "margin:3em auto;&:first-child{margin-top:0;}",
          ],
        },
        "displayName": "Section",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "Section-sc-1eulrsm-0",
        "target": "section",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <section
      className="c0"
    />
  </StyledComponent>
</Section>
`;
